tests/libsecret/secret.go

```diff
@@ -35,6 +40,27 @@ func New(name string, data map[string][]byte) *kubev1.Secret {
 				secretLabel: name,
 			},
 		},
-		Data: data,
+		Data:       data.Data(),
+		StringData: data.StringData(),
 	}
 }
+
+type DataBytes map[string][]byte
```

Comment:
It is never used...

Reply:
Do we accept dead code @AlonaKaplan ?

Reply:
Sometime there is a need to prepare ahead and provide the infra.

See usages here https://github.com/kubevirt/kubevirt/pull/11547
There is another one which is under work.